### PR TITLE
Deploy through azure web service

### PR DESCRIPTION
**Pull request by saipramod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `yarn start`
+### `yarn run dev`
```
Comment (lzl124631x) [inline README.md:7]: this change is unnecessary to me. I've been using `npm run dev` in other projects but `npm start` is also fine. I think we don't need to change it unless we have a strong reason. It's for now consistent with backend.

File: package.json [hunk 2]
```diff
@@ -12,11 +12,11 @@
         "react-scripts": "^3.4.1"
     },
     "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
+        "dev": "react-scripts start",
+        "build": "set REACT_APP_API_PROD=https://covidprojectionsapi.azurewebsites.net && react-scripts build",
```
Comment (lzl124631x) [inline package.json:16]: I would not hardcode the env like this because this env is subject to change.

File: src/service.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { StackedChartData } from "./stackedchart/StackedChartData";
 import { MapData } from "./type";
 import { TimeSeriesData } from "./projection/type";
 
-const backendUrl = `${window.location.protocol}//${window.location.hostname}:6789`;
+const backendUrl = process.env.REACT_APP_API_PROD != null ? process.env.REACT_APP_API_PROD : `${window.location.protocol}//${window.location.hostname}:6789`;
```
Comment (lzl124631x) [inline src/service.ts:6]: `process.env.API_ENDPOINT`

Comment (saipramod) [reply src/service.ts:6]: react only picks up variables with prefix `REACT_APP`

**Comment by saipramod** [comment]
I think much of your earlier comments have been taken care of, merging.

**Pull request merged**